x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/pipeline.tsx

```diff
@@ -44,8 +50,43 @@ const fieldsConfig: FieldsConfig = {
       },
     ],
   },
+  /* Optional field configs */
+  ignore_missing_pipeline: {
```

Comment:
I think we can use the common ignore missing field instead of re-declaring it here, you can find its implementation [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/common_fields/ignore_missing_field.tsx)